x/authz/spec/01_concepts.md

```diff
@@ -4,8 +4,14 @@ order: 1
 
 # Concepts
 
-## Authorization
-Any concrete type of authorization defined in the `x/authz` module must fulfill the `Authorization` interface outlined below. Authorizations determine exactly what privileges are granted. They are extensible and can be defined for any Msg service method even outside of the module where the Msg method is defined. Authorizations use the new `ServiceMsg` type from [ADR 031](../../../architecture/adr-031-msg-service.md).
+## Authorization and Grant
+`x/authz` module defines interfaces and messages grant authorizations to perform actions
+on behalf of one account to other accounts. The design is defined in the [ADR 030](../../../architecture/adr-030-authz-module.md).
+
+Grant is an allowance to execute an Msg by grantee address on behalf of the granter.
+Authorization is an interface which must be implemented by a concrete authorization logic to validate and execute grants. They are extensible and can be defined for any Msg service method even outside of the module where the Msg method is defined. See the `SendAuthorization` example below for more details.
+
+Authorizations use the new `Msg` type from [ADR 031](../../../architecture/adr-031-msg-service.md).
 
 
 +++ https://github.com/cosmos/cosmos-sdk/blob/c95de9c4177442dee4c69d96917efc955b5d19d9/x/authz/types/authorizations.go#L15-L24
```

Comment:
Snippet should be updated

Reply:
Yes, I created a task to update this after the merge: https://github.com/cosmos/cosmos-sdk/issues/9257. Thanks.